spec/support/api/schemas/songs.json

```diff
@@ -0,0 +1,12 @@
+{
+  "songs": [
+    {
+      "id": "870fd74a-908a-4705-8d98-2677df49ea0d",
+      "name": "Michael",
+      "artist": "Jackson",
+      "spotify_uri": "safdsf",
+      "created_at": "2015-06-09T21:28:54Z",
+      "updated_at": "2015-06-09T21:28:54Z"
+    }
+  ]
+}
```

Comment:
Ah, so here you don't want to put "actual" JSON, but JSON Schema.

Go to http://jsonschema.net/ & paste in this JSON and it will spit out the (more or less) correct JSON schema you'll want in this file.

We'll talk more about JSON Schema in the JSON Structure lecture & elsewhere but if you want to read up on it you can read here: http://spacetelescope.github.io/understanding-json-schema/

Also, need a blank line at the end of this file or git gets annoyed :)